src/herbpy/tsr/block.py

```diff
@@ -61,25 +61,25 @@ def block_at_pose(robot, block, position, manip=None):
 
     # Block on the object
     T0_w = numpy.eye(4)
-    T0_w[:3,3] = position
+    T0_w[:3, 3] = position
 
-    Bw = numpy.zeros((6,2))
-    Bw[5,:] = [-numpy.pi, numpy.pi]
+    Bw = numpy.zeros((6, 2))
+    Bw[5, :] = [-numpy.pi, numpy.pi]
 
-    place_tsr = TSR(T0_w = T0_w,
-                    Tw_e = numpy.eye(4),
-                    Bw = Bw,
-                    manip = manip_idx)
+    place_tsr = TSR(T0_w=T0_w,
+                    Tw_e=numpy.eye(4),
+                    Bw=Bw,
+                    manip=manip_idx)
 
     ee_in_block = numpy.dot(numpy.linalg.inv(block.GetTransform()), manip.GetEndEffectorTransform())
-    ee_tsr = TSR(T0_w = numpy.eye(4), #ignored
-                 Tw_e = ee_in_block,
-                 Bw = numpy.zeros((6,2)),
-                 manip = manip_idx)
+    ee_tsr = TSR(T0_w=numpy.eye(4), #ignored
+                 Tw_e=ee_in_block,
+                 Bw=numpy.zeros((6, 2)),
+                 manip=manip_idx)
 
     place_tsr_chain = TSRChain(sample_start=False, 
-                               sample_goal = True,
-                               TSRs = [place_tsr, ee_tsr])
+                               sample_goal=True,
+                               TSRs=[place_tsr, ee_tsr])
     return [place_tsr_chain]
 
 @TSRFactory('herb', 'block', 'place_on')        
```

Comment:
I think this is one of the better ones I've seen, you can give it a shot and check the diff (make sure you are all pushed on the off chance the `.git` files get messed up):
https://gist.githubusercontent.com/dpaluy/3690668/raw/585de899ed37a8ff95e0f2d4137e2e82e463a71c/git.sh